samples/snippets/src/main/java/com/example/firestore/beam/README.md

```diff
@@ -11,21 +11,20 @@ TEMP_LOCATION=gs://<your-temp-bucket>/temp/
 NUM_WORKERS=<num-workers>
 MAX_NUM_WORKERS=<max-num-workers>
 ```
-2. Run the command to write example "cities" data:
+2. Run the command to write example data into the collection, "cities-beam-sample":
 ```
-mvn compile exec:java -Dexec.mainClass=ExampleFirestoreBeamWrite \
+mvn compile exec:java@beam-sample -Dexec.mainClass=com.example.firestore.beam.ExampleFirestoreBeamWrite \
```

Comment:
(just verifying)
This is `@beam-sample` yeah? and not `@qucikstart`?

Reply:
that's correct! We want to use the `beam-sample` id for these.